script/Ciel/Day 11/11_01_CIEL11_1C.txt

```diff
@@ -322,12 +341,15 @@ I breathe out a sigh.
 [sha:e019cbe6c0011ae28c7d80d66bdedb0978c6508a]{
 -- Page 72, Offset 29205.
 -- ロビーに椅子を置いても仕方がないだろう。
-There's no point in putting a chair in the lobby.
+There's no point in putting another chair in the lobby.
 }
 [sha:2090344807711b28dbbe3a880b3c971d61d281cd]{
 -- Page 72, Offset 29206.
 -- ロビーに椅子は有り余ってる。あるものを余計においてどうする。無駄に数が多いのは目障りだとなぜ分からない。まったくもってセンスがない。慎みがない。使い途がない。
-There are too many chairs in the lobby. What do they intend to do with these unnecessary things? Why don't they understand that too much is an eyesore? How absolutely tasteless. How arrogant. How useless.
+There are too many chairs in the lobby.
+ What is the point of having a surplus?
+ Do they not understand that it's an eyesore to have all these redundant objects?
+ How absolutely tasteless. How arrogant. How useless.
```

Comment:
Hmmmm, the first one should have a more direct direction question aspect to it, and I think 'eyesore' should probably be placed last
The second feels a bit lacking it's "incredulousness"
>How does she not understand that this ridiculous excess is an eyesore?
>Good lord, what lack of taste; of modesty; of purpose.